ieeeuoft/newhacks-site-2022
# NH-12: Add registration fields provided from Logistics

## Overview

- Resolves #NH-12
- Added fields to registration accordingly to the Logistics team document(DM for details)


## Unit Tests Created

- added test for optional fields


## Steps to QA

- edit `REGISTRATION_OPEN_DATE` __init__.py file in hackathon_site folder to access application
- make sure to run python manage.py migrate
